spec/support/acceptance/command_shell/linux.rb

```diff
@@ -0,0 +1,158 @@
+module Acceptance::CommandShell
+  LINUX = {
+    payloads: [
+      {
+        name: "cmd/unix/reverse_bash",
+        extension: "",
+        platforms: [:linux],
+        executable: true,
+        execute_cmd: ["${payload_path}"],
```

Comment:
Would you mind double checking that this fails when invoked through the shell like

```
        execute_cmd: ["sh ${payload_path}"],
```